lib/workers/repository/updates/generate.ts

```diff
@@ -388,9 +388,15 @@ export function generateBranchConfig(
     .map(getTableValues)
     .filter((x): x is string[] => is.array(x, is.string));
   if (tableRows.length) {
-    let table: string[][] = [];
+    const table: string[][] = [];
     table.push(['datasource', 'package', 'from', 'to']);
-    table = table.concat(tableRows!);
+    const seenRows: Set<string> = new Set();
+    tableRows.forEach((row) => {
+      if (!seenRows.has(row.join())) {
+        seenRows.add(row.join());
```

Comment:
This would incorrectly drop rows where we update the same datasource+package from 0.12 to 3.4 and also update from 0.1 to 23.4.

Reply:
why do you think that? 🤔 the join should be different for those rows

Reply:
I (I know now) incorrectly believed that it would join the elements without any separator.

Still, if you'd take exotic enough versions, I believe you can get this new piece of code to incorrectly dismiss entries.

Reply:
ok, so we should stringify to JSON instead. this should solve the rare conflicts